apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/inventory-dashboard.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+@Component({
+    selector: 'inventory-dashboard',
+    templateUrl: './inventory-dashboard.component.html',
+    styleUrls: ['./inventory-dashboard.component.scss'],
+
+})
+
+export class InventoryDashboardComponent implements OnInit {
+
+    constructor(private _router: Router) {
```

Comment:
Remove `private _router: Router` as it is unused
```suggestion
    constructor() {
```